recipes/export2graphlan/meta.yaml

```diff
@@ -20,5 +20,11 @@ requirements:
       - scipy
       - numpy
 test:
+  requires:
```

Comment:
Lets see what travis is saying. The test requirements are used for example to depend on `nose`, so testing specific dependencies.